app/src/validators/common.js

```diff
@@ -45,15 +45,27 @@ const type = {
     version: 'uuidv4'
   }),
 
-  metadata: (minKeyCount = 0, minValueStringLength = 0) => Joi.object()
+  /**
+   * custom metadata (object) type schema with parameters
+   * @param minKeyCount minimum number of tags allowed
+   * @param minValueStringLength minimum string length of tag value allowed
+   * @param maxKeyCount maximum number of tags allowed (default ot 9 because COMS also adds a `coms-id` tag by default)
+   */
+  metadata: ({ minKeyCount = 0, minValueStringLength = 1 } = {}) => Joi.object()
```

Comment:
now that we are making meta params optional for addMetadata and replaceMetadata operations, these arguments are kind of redundant. 

same for tagset().. but they could be useful some time in the future. oh well